src/js/users/components/CreateUser.tsx

```diff
@@ -15,40 +14,39 @@ type NewUser = {
     forceReset: boolean;
 };
 
+type CreateUserProps = {
+    /** Indicates whether the modal for creating a user is visible */
+    show: boolean;
+    /** The history object */
+    history: HistoryType;
+};
+
 /**
  * A dialog for creating a new user
  */
-export default function CreateUser() {
+export default function CreateUser({ show, history }: CreateUserProps) {
     const mutation = useMutation(create, {
         onSuccess: () => {
-            history.replaceState({ createUser: false }, "");
+            history.push({ state: { createUser: false } });
```

Comment:
Hmm why the change here? I feel like the use of replace state may have been deliberate to prevent having the modal on the history stack